### ISSUE TITLE
Is this result correct or not?

### ISSUE THREAD

**Issue by ly-jq** [VainF/DeepLabV3Plus-Pytorch#106]
After training the deeplab3plus resnet101 model with 30000 iters, the mean accuracy achieves to 0.678442 and miou 0.593765. This result is lower for your conclusion in the Cityscapes dataset, it is correct or not. I didn't use pretrained weight to train.
[deeplab3p.log](https://github.com/VainF/DeepLabV3Plus-Pytorch/files/11075732/deeplab3p.log)

**Comment by VainF** [comment]
Hi @ly-jq, could you please provide the training configs? The deeplabv3plus_resnet101 was contributed by a community user.

**Comment by haidrbilal** [comment]
According to encoder decoder atrous separable conv paper, they achieved 82.1% test set accuracy. What hyperparameters did they use while training?

**Comment by DC-SegNet** [comment]
> Hi @ly-jq, could you please provide the training configs? The deeplabv3plus_resnet101 was contributed by a community user.

Has the problem been solved?

**Comment by wmx2000219** [comment]
> Hi @ly-jq, could you please provide the training configs? The deeplabv3plus_resnet101 was contributed by a community user.
Hi, I recently trained deeplabv3+Xception on the VOC2012 dataset, and after 150 epochs the miou metric on the validation set was only 0.26, and the acc metric had 0.72. I don't know what's wrong with the code, is it because of the small number of epochs trained?
